src/schema/schema.graphql

```diff
@@ -317,7 +319,38 @@ type DtTimeInfo {
 	timezone: String # tz
 }
 
-type CalendarItemHitInfo {
+type Invitation { # <inv>
+	type: String!
+	sequenceNumber: Int! # seq
+	id: Int!
+	componentNum: Int! # compNum
+	recurrenceId: String # recurId
+	tz: CalTZInfo
+	components: [InviteComponent]! # comp
+	mimeParts: MimePart
+}
+
+type CalTZInfo { # <tz>
```

Comment:
These items `CalTZInfo` and `TzOnsetInfo` aren't required yet, but we will likely be using them soon so I left them in.